js/payment-methods/bitcoin-lightning.js

```diff
@@ -17,10 +17,23 @@ app.paymentMethods.bitcoinLightning = (function() {
 		// Used internally to reference itself:
 		ref: 'bitcoinLightning',
 
+		generalInfo: {
```

Comment:
Instead of `getInfo` object, this can simply be `description` with long text (think paragraph potentially).